app/views/contacts/edit.html.haml

```diff
@@ -0,0 +1,20 @@
+.row.justify-content-center
+  %h2.title= t('.contacts_edit')
+.row
+  .col-lg-5.col-md-5
+    = simple_form_for @contact, as: @contact do |f|
+      = f.input :email, label: 'Email', autofocus: true, placeholder: 'example@mail.com'
+      = f.input :phone, label: t('.phone')
```

Comment:
since you have a format validation, add placeholder with the sample of the phone